tests/testdata_gravity/grav_subs_points.yml

```diff
@@ -0,0 +1,16 @@
+# Configuration file for use with grav_subs_points.py
+
+input:
+  diffdates:
+    - [2020-07-01, 2018-01-01]
+    
+stations:
+  grav:
+    '2020_2018': /project/fmu/users/tral/subscript/subscript/tests/testdata_gravity/station_coordinates.csv
```

Comment:
I guess this configuration file has not been used for testing. But still it will be good to keep it as relative path.

Reply:
You are right, this config was added to the test dataset, but will be used in a job not implemented yet. But will update to relative path now.